templates/security/cves/_cve-card.html

```diff
@@ -28,40 +28,24 @@ <h2 class="p-heading--5">
           </span>
         </div>
       </div>
-      <div class="col-3 col-medium-2">
-        {% if cve.summarized_status %}
-          {% set status = cve.summarized_status %}
-          {% if status.name == "Some fixed" %}
-            <p>
-              Some fixes available <span class="u-text--muted">{{ status.fixed_count }} of {{ status.total_count }}</span>
-            </p>
-          {% elif status.name == "Needs evaluation" %}
-            <p>
-              <i class="p-icon--help"></i> <span>Needs evaluation</span>
-            </p>
-          {% elif status.name == "Fixed" %}
-            <p>
-              <i class="p-icon--success"></i> <span>Fixed</span>
-            </p>
-          {% elif status.name == "Vulnerable" %}
-            <p>
-              <i class="p-icon--warning"></i> <span>Vulnerable</span>
-            </p>
-          {% elif status.name == "DNE" %}
-            <p>
-              <span>Not in release</span>
-            </p>
-          {% else %}
-            {{ status }}
-          {% endif %}
-        {% endif %}
-      </div>
+      {% if cve.summarized_status %}
+        {% set status = cve.summarized_status %}
+        <div class="col-3 cve-summary col-medium-2" {% if status.icon %}style="padding-top: 0.4rem;"{% endif %}>
```

Comment:
The missing of icon in "Not in release" excludes top padding, making it look inconsistent compared to the other statuses with icons
<img width="956" alt="Screenshot 2024-08-07 at 4 22 52 PM" src="https://github.com/user-attachments/assets/30eaf0c1-0729-400c-8a4c-32f46b0415ba">

Reply:
Could've sworn I'd set it to check for the icon, fixed!